pages/common/pio-home.md

```diff
@@ -0,0 +1,28 @@
+# pio home
+
+> Launch the PlatformIO Home web server.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_home.html>.
+
+- Open PlatformIO Home in your default web browser:
```

Comment:
We try to avoid `your`, `you`, `her`, etc.
```suggestion
- Open PlatformIO Home in the default web browser:
```

Reply:
(`your`, `you`, `her`, etc. = personal pronouns 😉)